### ISSUE TITLE
[BUG] Rift Explorer perma refreshing loop

### ISSUE THREAD

**Issue by zaryar** [Pupix/rift-explorer#151]
I can start Rift Explorer 7 and everything looks good, but if I can't click anything because the Interface keeps refreshing. The cmd permanently puts out: ```
[1] Lockfile information sent; clearing riotclient check interval
[1] lcuswaggeren
```
And my console puts out: ```
Connected to league client! | instrument.ts:129
triggered swagger refresh | instrument.ts:129
```

![Screenshot_1](https://user-images.githubusercontent.com/46141276/131392252-0d81e8c8-df38-4c42-83cf-4f7751b1d1ca.png)

**Comment by kevscript** [comment]
I'm experiencing the same issue.

**Comment by whiitehead** [comment]
Still no word on this, eh?

**Comment by Daniel41920** [comment]
> Still no word on this, eh?

No, and there will never be any word because the program can't work anymore. The riot client disables the flag that enables the league client to give endpoint information. Unless you can authenticate without the riot client or can stop the riot client from doing it, have fun reverse engineering LeagueClient.exe.
Just use a site that has the endpoints for you:
http://www.mingweisamuel.com/lcu-schema/tool/
https://lcu.vivide.re/

**Comment by Hi-Ray** [comment]
> No, and there will never be any word because the program can't work anymore. The riot client disables the flag that enables the league client to give endpoint information. Unless you can authenticate without the riot client or can stop the riot client from doing it, have fun reverse engineering LeagueClient.exe.

Not quite however the issue rift-explorer has stopped development is due to personal issues as well as  the developers working on other projects, I can't promise anything but there may be a fix in the future I haven't been able to replicate this bug but I'll try fix this if I have the time.

**Comment by Hi-Ray** [comment]
If you really need it I'd advise using mingwei's implementation as that is something i have helped with to keep up to date and to my knowledge is up to date
